Cython/Compiler/ExprNodes.py

```diff
@@ -12861,7 +12861,10 @@ def generate_evaluation_code(self, code):
         code.mark_pos(self.pos)
         self.allocate_temp_result(code)
         self.test.generate_evaluation_code(code)
-        code.putln("if (%s) {" % self.test.result())
+        cond_val = self.test.result()
+        if not cond_val.startswith("(") or not cond_val.endswith(")"):
+            cond_val = "(" + cond_val + ")"
```

Comment:
```suggestion
            cond_val = "(%s)" % cond_val
```

Just prefer to avoid string concatenation (at least when there's more than two parts)